src/__tests__/Settlements.ts

```diff
@@ -0,0 +1,63 @@
+import BigNumber from 'bignumber.js';
+
+import { Context, Venue } from '~/internal';
+import { Settlements } from '~/Settlements';
+import { dsMockUtils, entityMockUtils } from '~/testUtils/mocks';
+import { Mocked } from '~/testUtils/types';
+
+jest.mock(
+  '~/api/entities/Venue',
+  require('~/testUtils/mocks/entities').mockVenueModule('~/api/entities/Venue')
+);
+
+describe('Claims Class', () => {
+  let context: Mocked<Context>;
+  let settlements: Settlements;
+
+  beforeAll(() => {
+    dsMockUtils.initMocks();
+    entityMockUtils.initMocks();
+  });
+
+  beforeEach(() => {
+    context = dsMockUtils.getContextInstance();
+    settlements = new Settlements(context);
+  });
+
+  afterEach(() => {
+    dsMockUtils.reset();
+    entityMockUtils.reset();
+  });
+
+  afterAll(() => {
+    dsMockUtils.cleanup();
+    entityMockUtils.cleanup();
+  });
+
+  describe('method: getVenue', () => {
+    test('should return a Venue by its id', async () => {
+      const venueId = 1;
+      const matchingVenue = new Venue({ id: new BigNumber(venueId) }, context);
```

Comment:
The convention is to use entityMockUtils.getVenueInstance({ id }). We try to avoid instancing entities in tests because it has caused problems in the past